scripts/dev_changelog.py

```diff
@@ -64,12 +64,12 @@
     "deprecation": "Deprecated classes and functions",
     "new feature": "New features",
     "enhancement": "Enhancements",
-    "bug": "Fixed bugs",
+    "bugfix": "Fixed bugs",
```

Comment:
Should we have both entries in order to be backwards compatible? I guess we could also simply change all old PRs tagged with "bug" to "bugfix", alternatively.

Reply:
I believe I already changed all past instances of merged PRs with "bug" -> "bugfix" by renaming the "bug" label to "bugfix". This led to an issue in the Issues page where all issues with the "bug" label were also renamed to "bugfix". To fix that issue, I recreated the "bug" label and used the issues page's search/filter to select all issues with the "bugfix" label then I simply relabeled them all in one go back to "bug".

Is there a PR I missed?

Reply:
In that case: great, no further comments. :-)